src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ActiveDirectoryAuthenticationProvider.cs

```diff
@@ -582,6 +582,10 @@ private static TokenCredentialData CreateTokenCredentialInstance(TokenCredential
                     defaultAzureCredentialOptions.WorkloadIdentityClientId = tokenCredentialKey._clientId;
                 }
 
+                // This is library code that is not deployed, so we can default
+                // to DefaultAzureCredential.  Applications that use our library
+                // are responsible for replacing it with a more secure option.
```

Comment:
Please adjust it with below comment.

// SqlClient is a library and provides support to acquire access token using 'DefaultAzureCredential' on user demand when they specify 'Authentication = Active Directory Default' in connection string.